packages/interaction/src/InteractionTrackingData.js

```diff
@@ -121,11 +121,28 @@ export default class InteractionTrackingData
     {
         this._doSet(this.constructor.FLAGS.LEFT_DOWN, yn);
     }
+
+    /**
+     * Did the user touch down in the DisplayObject?
+     *
+     * @private
+     * @member {boolean}
+     */
+    get touchDown()
+    {
+        return (this._flags & this.constructor.FLAGS.TOUCH_DOWN) !== 0;
+    }
+
+    set touchDown(yn) // eslint-disable-line require-jsdoc
+    {
+        this._doSet(this.constructor.FLAGS.TOUCH_DOWN, yn);
+    }
 }
 
 InteractionTrackingData.FLAGS = Object.freeze({
     NONE: 0,
     OVER: 1 << 0,
     LEFT_DOWN: 1 << 1,
     RIGHT_DOWN: 1 << 2,
+    TOUCH_DOWN: 1 << 3,
```

Comment:
Because the instance of the IntractionTrackingData is attached to an ID for a touch, I think that we should be able to stick with `LEFT_DOWN`. If you feel that is confusing in the code, perhaps adding `primaryDown` as an alias for `leftDown` (which makes more sense for pen friendly code anyway)?